site/content/resources/upgrading.md

```diff
@@ -15,6 +15,14 @@ Contour currently only tests sequential upgrades, i.e. without skipping any mino
 This approach is recommended for users in order to minimize downtime and avoid any potential issues.
 If you choose to skip versions while upgrading, please note that this may lead to additional downtime.
 
+# Known issues
+
+1. Envoy pod stuck in pending state
+
+    If number of envoy instances is no less than number of kubernetes nodes in the clusters, during rolling upgrade, new envoy pod will be stuck in pending stage because old envoy pod is occupying host pod.
```

Comment:
```suggestion
    If number of envoy instances is no less than number of kubernetes nodes in the clusters, during rolling upgrade, new envoy pod will be stuck in pending stage because old envoy pod is occupying host port.
```